pennylane/labs/__init__.py

```diff
@@ -27,7 +27,7 @@
 
     dla
     resource_estimation
-
+    vibrational
```

Comment:
There is nothing showing inside this in the docs. Please remove it and we will add it in a a following PR.